core/parameters/src/view.rs

```diff
@@ -611,12 +611,12 @@ impl From<ExtCostsConfigView> for crate::ExtCostsConfig {
 }
 
 #[cfg(test)]
+#[cfg(not(feature = "nightly"))]
+#[cfg(not(feature = "statelessnet_protocol"))]
```

Comment:
FWIW I still prefer `ignore` as it allows the test runner to see the total number of tests and indicate which ones have been ignored, but its not serious enough to block the PR over it.

Reply:
Will keep in mind and change in some future random PR!